apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -147,6 +148,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
         description={t("booking_questions_description")}
         addFieldLabel={t("add_a_booking_question")}
         formProp="bookingFields"
+        eventTypeLocationsLength={eventTypeLocationsLength}
```

Comment:
Can we save FormBuilder to having to know more than needed.

```suggestion
        shouldDisplayLocation ={eventType.locations.length > 1}
```